### PR TITLE
Replace moment usage

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Replace moment.js on the admin react project](https://trello.com/c/7CTMa98U/1095-replace-momentjs-on-the-admin-react-project)

File: lib/admin/server.js [hunk 1]
```diff
@@ -37,8 +37,8 @@ function machinesLastPing () {
 
       if (downRows.length === 1) {
         const row = downRows[0]
-        const age = moment.duration(row.age, 'seconds')
-        return `${row.name} down for ${age.humanize()}`
+        const age = intervalToDuration({ start: 0, end: secondsToMilliseconds(row.age) })
+        return `${row.name} down for ${formatDuration(age)}`
```
Comment (josepfo) [inline lib/admin/server.js:41]: Are we displaying the duration in milliseconds or seconds?

File: new-lamassu-admin/src/utils/timezone-list.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+  'America/St_Johns': 'Newfoundland and Labrador',
+  'America/Sao_Paulo': 'Brasilia',
+  'America/Tijuana': 'Tijuana',
+  'America/Montevideo': 'Montevideo',
+  'America/Argentina/Buenos_Aires': 'Buenos Aires, Georgetown',
+  'America/Godthab': 'Greenland',
+  'America/Los_Angeles': 'Pacific Time',
+  'Atlantic/Azores': 'Azores',
+  'Atlantic/Cape_Verde': 'Cape Verde Islands',
+  GMT: 'UTC',
```
Comment (josepfo) [inline new-lamassu-admin/src/utils/timezone-list.js:30]: Missing ' ?

File: new-lamassu-admin/src/utils/timezone-list.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
+import { intervalToDuration } from 'date-fns'
+import { getTimezoneOffset } from 'date-fns-tz'
+import * as R from 'ramda'
+
+const timezones = {
```
Comment (josepfo) [inline new-lamassu-admin/src/utils/timezone-list.js:5]: These labels are supposed to improve UX, right?

**Review by josepfo — CHANGES_REQUESTED** [review]

**Review by josepfo — APPROVED** [review]

**Pull request merged**